extensions/ql-vscode/src/query-history.ts

```diff
@@ -511,7 +532,9 @@ export class QueryHistoryManager {
   private async tryOpenExternalFile(fileLocation: string) {
     const uri = vscode.Uri.file(fileLocation);
     try {
-      await vscode.window.showTextDocument(uri);
+      await vscode.window.showTextDocument(uri, {
+        preview: false
```

Comment:
Is this also used by the source archive? If so it's a behaviour change worth documenting.

Reply:
This is a minor change, but my thinking is that if someone went to the trouble of opening a log file or DIL file, then they probably don't want it to be stuck in preview mode and disappear when navigating around. 

Is this a change that users would want to know about? My feeling is that this change just makes things _work_ better.  Knowing about it won't change users' behaviours. Mentioning it in docs or changelogs might just be noise. Only reason to do it IMO is if we want recognition that there is a snazzy new feature.

Reply:
It makes complete sense for opening DIL (or even opening queries). I was curious about whether this handler is shared by the code that opens source archive files -- there it might become quite cluttered if we open every source link in a permanent tab. (I personally would not mind, but it's easy to click on a lot of result links.)

Reply:
Oh...I see what you're asking. No. This code path is only used for commands coming from query history. So, user needs to right-click and actively try to see this file.